frontend/src/metabase/reference/components/EditHeader.jsx

```diff
@@ -15,7 +16,7 @@ const EditHeader = ({
   onSubmit,
   revisionMessageFormField,
 }) => (
-  <div className={cx("EditHeader wrapper py1 px3", S.editHeader)}>
+  <div className={cx("EditHeader", CS.wrapper, "py1 px3", S.editHeader)}>
```

Comment:
2 things here.
1. `EditHeader` doesn't seem to be used anywhere. Could you double check and possibly remove it if that's the case?
2. Should you use `CS.py1` and `CS.px3` too?